.circleci/config.yml

```diff
@@ -0,0 +1,85 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+       - image: circleci/ruby:2.3.7
+         environment:
+          # - COVERAGE_REPORTER_CIRCLECI_TOKEN: 6a18d56f89f0c64450823b313d17b6359ac0879a
+          # - COVERAGE_REPORTER_VCS_TOKEN: 36cf237082d794c1e48088a986d8eff7bede0e7f
+          - DATABASE_URL: postgres://postgres:postgres@localhost:5432
+       - image: postgres:alpine
+         environment:
+           - POSTGRES_USER: postgres
+           - POSTGRES_PASSWORD: postgres
+           - POSTGRES_DB: acts_as_bookable_test
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "Gemfile.lock" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run:
+          name: Download cc-test-reporter
+          command: |
+            mkdir -p tmp/
+            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./tmp/cc-test-reporter
+            chmod +x ./tmp/cc-test-reporter
+
+      - run: mv .circleci/database.ci.yml config/database.yml
+
+      - run:
+          name: install dependencies
+          command: |
+            bundle install --jobs=4 --retry=3 --path vendor/bundle
+      - save_cache:
+          paths:
+            - ./vendor/bundle
+          key: v1-dependencies-{{ checksum "Gemfile.lock" }}
+
+      # # Database setup
+      # - run: bundle exec rake db:create
+      # - run: bundle exec rake db:schema:load
+
+      # run tests!
+      - run:
+          name: run tests
+          command: |
+            mkdir /tmp/test-results
+            TEST_FILES="$(circleci tests glob "spec/**/*_spec.rb" | circleci tests split --split-by=timings)"
+
+            bundle exec rspec --format progress \
+                            --format RspecJunitFormatter \
```

Comment:
You'll need to add this as a dev dependency in the gemspec

Reply:
done